openfisca_core/tests/assets/variable_parameters.yaml

```diff
@@ -0,0 +1,99 @@
+# Use the VAR node and its condition children
+# to avoid using variations of parameter names to express conditions,
+# and compute the condition in your formula.
+
+# This is very useful for "terminal variations",
+# that is variations that happen close to the leaf nodes of the param tree
+
+# conditions use real variable names that you will pass from your formula
+# TODO or should we compute them for you ? Ideally yes, but what about period conversions ?
+
+# Regular value parameters :
+
+- variable: fillon_taux_max
+  description: Taux maximum de l'allègement général
+  format: percent
+  VAR:
+    - condition: "20 <= effectif_entreprise"
+      VALUES:
+        2018-01-01: .2850
+        2017-01-01: .2850
+        2016-01-01: .2842
+        2015-01-01: .2835
+        2005-07-01: .26
+    - condition: "effectif_entreprise < 20"
```

Comment:
Yes, that's why : 
- I'd like to review a domain (*prélèvements sociaux* because I know it well) and draw conclusions
- Possibly limit conditions to simple statements, following this logic : *if the currently available parameter models can't absorb the complexity of that formula, just use simple VALUE parameter (for dates) and write an ad-hoc formula*